easybuild/easyconfigs/r/RELION/RELION-5.0.0_beta.2024.05.03-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,253 @@
+]
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('poetry', '1.7.1'),  # use poetry compatible with build 1.x
+    ('meson-python', '0.15.0'),  # pandas 2.1.4 needs meson >= 1.2.1
+]
+
+dependencies = [
+    # required dependencies #
+    ('CUDA', '12.1.1', '', SYSTEM),
+    ('LibTIFF', '4.5.0'),
+    ('tbb', '2021.11.0'),
+    ('libpng', '1.6.39'),
+    ('morphosamplers', '0.0.10'),   # required for: src/tomography_python_programs/get_particle_poses/filaments.py
+
+    # optional dependencies #
+    ('tcsh', '6.24.10'),
+    ('gnuplot', '5.4.8'),
+    ('FLTK', '1.3.8'),            # for GUI
+    ('X11', '20230603'),          # for GUI
+    ('PBZIP2', '1.1.13'),         # for compression
+    ('zstd', '1.5.5'),            # for compression
+    ('XZ', '5.4.2'),              # for compression
+    ('Ghostscript', '10.01.2'),   # for PDF files
+    ('xpdf', '4.04'),             # pdf viewer
+    # ('evince', '45.0'),         # pdf viewer
+    # ('ctffind', '4.1.14'),      # lens defocus estimation
+    # ('AreTomo2', '1.0.0', versionsuffix),     # alternative movie alignment
+    # ('MotionCor2', '1.5.0'),                  # alternative movie alignment
+    # ('MotionCor3', '1.0.1', versionsuffix),   # alternative movie alignment
+    # ('ResMap', '1.1.4'),        # alternative local resolution estimation
+
+    # dependencies from conda environment.yaml file #
+    ('Python', '3.11.3'),
+    ('PyTorch-bundle', '2.1.2', versionsuffix),
+    ('SciPy-bundle', '2023.07'),
+    ('tqdm', '4.66.1'),
+    ('scikit-learn', '1.3.1'),
+    ('matplotlib', '3.7.2'),
+    ('PyQt5', '5.15.10'),
+    ('Biopython', '1.83'),
+    ('Seaborn', '0.13.2'),
+    ('dill', '0.3.7'),
+    ('umap-learn', '0.5.5'),
+    ('einops', '0.7.0'),
+    ('mrcfile', '1.5.0'),
+    ('t-SNE-CUDA', '3.0.1', versionsuffix),
+    ('ESM-2', '2.0.0', versionsuffix),
+    ('topaz', '0.2.5.20231120', versionsuffix),
+    ('dask', '2023.9.2'),
+    ('imageio', '2.33.1'),
+    ('napari', '0.4.19.post1'),
+    ('tensorboard', '2.15.1'),
+]
+
+# set this to the path to the Blush model weight database
+# workaround until we merged the "Dataset" easyblock:
+#   https://github.com/easybuilders/easybuild-easyblocks/pull/3246
+#   https://github.com/easybuilders/easybuild-framework/pull/4474
+_torch_home = ''
+
+_copts = [
+    '-DCMAKE_SHARED_LINKER="$LIBS"',
+    '-DMPI_INCLUDE_PATH="$MPI_INC_DIR"',
+    '-DCUDA_NVCC_FLAGS="--std=c++14"',
+    '-DCUDA_ARCH="%(cuda_cc_semicolon_sep)s"',
+    '-DCUDA_TEXTURE=ON',
+    '-DDoublePrec_CPU=OFF',
+    '-DFETCH_WEIGHTS=OFF',
+    '-DTORCH_HOME="%s"' % _torch_home,
+]
+
+configopts = ' '.join(_copts)
+
+install_cmd = 'make -j %(parallel)s install'
+
+exts_defaultclass = 'PythonPackage'
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'use_pip': True,
+    'download_dep_fail': True,
+    'sanity_pip_check': True,
+}
+
+exts_list = [
+    ('pyhmmer', '0.10.4', {
```

Comment:
@jfgrimm We are using PyHMMER 0.10.6 as a dependency built with Cython3 builddep at our site. See #20544

Reply:
thanks; merged that PR and switched to it in [1792cd9](https://github.com/easybuilders/easybuild-easyconfigs/pull/19678/commits/1792cd933c9ae457b0ad2b8c166234eaff870819)